src/cookbook/ScottPlot.Cookbook/Recipes/Plottable/RadialGauge.cs

```diff
@@ -0,0 +1,325 @@
+
+            var RadialGauge = plt.AddRadialGauge(values);
+            RadialGauge.GaugeLabelPos = 50;
+        }
+    }
+
+    public class RadialGaugeLabelFontPct : IRecipe
+    {
+        public string Category => "Plottable: RadialGauge";
+        public string ID => "radialgauge_labelfontpct";
+        public string Title => "Gauge label font percentage";
+        public string Description =>
+            "Size of the gague label text as a percentage of the gauge width. " +
+            "Values in the range [0-100], default value is 75 [percent].";
+
+        public void ExecuteRecipe(Plot plt)
+        {
+            plt.Palette = ScottPlot.Drawing.Palette.Nord;
+            double[] values = { 100, 80, 65, 45, 20 };
+
+            var RadialGauge = plt.AddRadialGauge(values);
+            RadialGauge.GaugeLabelsFontPct = 50;
+        }
+    }
+
+    public class RadialGaugeLabelColor : IRecipe
+    {
+        public string Category => "Plottable: RadialGauge";
+        public string ID => "radialgauge_labelcolor";
+        public string Title => "Gauge label color";
+        public string Description =>
+            "Color of the labels.";
+
+        public void ExecuteRecipe(Plot plt)
+        {
+            plt.Palette = ScottPlot.Drawing.Palette.Nord;
+            double[] values = { 100, 80, 65, 45, 20 };
+
+            var RadialGauge = plt.AddRadialGauge(values);
+            RadialGauge.GaugeLabelsColor = Color.Yellow;
+        }
+    }
+
+    public class RadialGaugeBackDim : IRecipe
+    {
+        public string Category => "Plottable: RadialGauge";
+        public string ID => "radialgauge_backdim";
+        public string Title => "Background gauges dim";
+        public string Description =>
+            "Dimmed percentage used to draw the gauges' background. Values in the range [0-100], default value is 90 [percent]. " +
+            "A value of 100 renders invisible the background gauges.";
+
+        public void ExecuteRecipe(Plot plt)
+        {
+            plt.Palette = ScottPlot.Drawing.Palette.Nord;
+            double[] values = { 100, 80, 65, 45, 20 };
+
+            var RadialGauge = plt.AddRadialGauge(values);
+            RadialGauge.DimPercentage = 50;
+        }
+    }
+
+    public class RadialGaugeBackNorm : IRecipe
+    {
+        public string Category => "Plottable: RadialGauge";
+        public string ID => "radialgauge_backnorm";
+        public string Title => "Background gauges normalization";
+        public string Description =>
+            "The gauges' background are drawn as full circles by default. If normalization is set to true, background gauges are limited to the values of StartingAngle and AngleRange.";
+
+        public void ExecuteRecipe(Plot plt)
+        {
+            plt.Palette = ScottPlot.Drawing.Palette.Nord;
+            double[] values = { 100, 80, 65, 45, 20 };
+
+            var RadialGauge = plt.AddRadialGauge(values);
+            RadialGauge.NormBackGauge = true;
```

Comment:
This is kind of a weird name. I think angle range should be nullable, and if it's null it's treated as if it's 360. If not, you use that number.

Reply:
I'm OK with changing the name to whatever feels more appropiate.
Making it nullable is a big modification since this would affect the `RadialGauge` plot code in several places.

Reply:
> Making it nullable is a big modification since this would affect the RadialGauge plot code in several places.

Eh, not really, your IDE should be able to show you everywhere it's referenced. And I think changing it now is going to be much less painful than changing it later.

Reply:
Still a work in progress but I renamed this to `CircularBackground`. Anything is probably okay as long as it has a good cookbook example and XML docs.